### DISCUSSION TITLE
直接从Release里面下载的mac版本已编译好的发行包，里面好像没有lib文件？

### DISCUSSION THREAD

**Discussion by HelloAllenZhu** [CefView/QCefView#206] [General]
直接从Release里面下载的mac版本已编译好的发行包，里面好像没有lib文件？

**Comment by tishion** [comment]
你可能需要补一下mac开发的基础知识？

**Reply by HelloAllenZhu** [reply]
我的意思是，直接在Qt里面导入mac的lib库文件，难道就是那个PlugIns/Chromium Embedded Framework.framework文件？
没搞明白QtCreator里面怎么使用XCode生成的静态库或者动态库、导入库。

**Reply by tishion** [reply]
QCefView.framework
